src/export-api-v2.ts

```diff
@@ -502,70 +600,400 @@ async function calcWearableSubmissionFeeUSD_fromSubgraph(manaUsd: number): Promi
+        toBlock: toBlockHexPolygon,
+        fromAddress: fromAddr,
+        toAddress: toAddr,
+        category: ['external', 'erc20'],
+        withMetadata: true,
+        maxCount: '0x3e8'
+      }
+    ]
+
+    try {
+      const result = await rpcPolygon<{ transfers?: AlchemyTransfer[] }>(
+        'alchemy_getAssetTransfers',
+        params as unknown[]
+      )
+      if (result?.transfers?.length) {
+        allTransfers.push(...result.transfers)
+      }
+    } catch (err: any) {
+      console.error(
+        '[WCC Payout POLYGON] alchemy_getAssetTransfers failed for',
+        fromAddr,
+        '->',
+        toAddr,
+        err?.message || String(err)
+      )
+    }
+  }
+}
+
+  const seen = new Set<string>()
+  let usdTotal = 0
+  let manaTotal = 0
+
+  for (const tr of allTransfers) {
+    const key = tr.uniqueId || `${tr.hash}:${tr.asset}:${tr.value}`
+    if (seen.has(key)) continue
+    seen.add(key)
+
+    const amount = Number(tr.value) || 0
+    const asset = tr.asset || ''
+
+    if (!amount || !asset) continue
+
+    const usd = tokenAmountToUsd(asset, amount, manaUsd)
+    usdTotal += usd
+
+    if ((asset || '').toUpperCase() === 'MANA') {
+      manaTotal += amount
+    }
+  }
+
+  const txCount = seen.size
+
+  console.log(
+   '[Wearable Curators Committee Payout]',
+    'txCount:',
+    txCount,
+    'manaTotal:',
+    manaTotal,
+    'usdTotal:',
+    usdTotal
+  )
+
+  return {
+    name: 'Wearable Curators Committee Payout',
+    description: 'Compensation paid from the DAO treasury to Wearable Curators Committee members (last 30 days)',
+    txCount,
+    manaTotal,
+    usdTotal
+  }
+}
+
+//---- other expense 
+async function calcOtherExpensesUSD(
+  startTs: number,
+  endTs: number,
+  manaUsd: number
+): Promise<IncomeRow> {
+  
+  if (!ALCHEMY_ETH_URL && !ALCHEMY_POLYGON_URL) {
+    console.warn('No Alchemy URLs are set, Other expenses will be 0')
+    return {
+      name: 'Other',
+      description:
+        'Other expenses from the DAO treasuries not categorized under specific committees (last 30 days)',
+      txCount: 0,
+      manaTotal: 0,
+      usdTotal: 0
+    }
+  }
+
+  const allTransfers: AlchemyTransfer[] = []
+
+  // ---- ETH ----
+  if (ALCHEMY_ETH_URL) {
+    const fromBlockNum = await findBlockAtOrBeforeTimestamp(startTs)
+    const toBlockNum = await findBlockAtOrBeforeTimestamp(endTs)
+    const fromBlockHex = intToHex(fromBlockNum)
+    const toBlockHex = intToHex(toBlockNum)
+
+    for (const fromAddr of DAO_TREASURY_ETH) {
+      console.log('[Other ETH] query', { fromAddr })
+      const params = [
+        {
+          fromBlock: fromBlockHex,
+          toBlock: toBlockHex,
+          fromAddress: fromAddr,
+          
+          category: ['external', 'erc20'],
+          withMetadata: true,
+          maxCount: '0x3e8'
+        }
+      ]
+
+      try {
+        const result = await rpc<{ transfers?: AlchemyTransfer[] }>(
+          'alchemy_getAssetTransfers',
```

Comment:
does this overlaps with line 703 ? https://github.com/Decentraland-DAO/transparency/pull/112/files#diff-4b91d1786ca951ef9de1f4be5bd2ed59c3b157b0e463399831dc09fee42b3ef9R703

is the same method from the same addrsss